tools/proofers/for_mentors.php

```diff
@@ -171,6 +174,101 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
     output_page_list_table($mentored_round, $projectid);
 }
 
+/**
+ * Checkout/return pages for a project.
+ *
+ * We do this per-project instead of at the top so that messages about the
+ * checkout are included with the project on the page.
+ */
+function checkout_return_project_pages(Round $mentored_round, string $projectid)
+{
+    // return if the user hasn't requested checkouts/returns for this project
+    if ((!isset($_POST["checkout_proofreader"]) && !isset($_POST["return_proofreader"]))
+        || get_projectID_param($_POST, "projectid") != $projectid) {
+        return;
+    }
+
+    // get the list of pages to checkout/return for the given user
+    $checkout_pages = [];
+    $return_pages = [];
+    if (isset($_POST["checkout_proofreader"])) {
+        // pages for proofreader
+        validate_projectID($projectid);
+        $sql = sprintf(
+            "
+            SELECT image
+            FROM $projectid
+            WHERE $mentored_round->user_column_name = '%s'
+                AND state like '%%avail'
```

Comment:
```suggestion
                AND state = '$page_state'
```
Not saying that there is anything wrong with this although it is arguable that the string stored in the database should be defined in only one place but since we have $page_state we might as well use it here.